Doc/library/binascii.rst

```diff
@@ -119,11 +119,6 @@ The :mod:`binascii` module defines the following functions:
       crc = binascii.crc32(b" world", crc)
       print('crc32 = {:#010x}'.format(crc))
 
-   .. versionchanged:: 3.0
-      The result is always unsigned.
-      To generate the same numeric value across all Python versions and
```

Comment:
These versionchanged notes were true so they should not be deleted. The only edit I'd make to be more modern is to be more direct and say "in older Python versions" instead of "across all Python versions".

historical note: The "all Python versions" text here and code suggestion was referring to people who needed code to work right on Python 2 (or 1.x) as well where the old <=2.7 fixed precision signed Python int type was returned. That type used a C long internally so it'd even differ between 32-bit and 64-bit builds in Python 2. We no longer need to cater to Python 2 users, but we generally don't delete versionchanged info from our docs, even when old.

Reply:
I have read [issue1202](https://bugs.python.org/issue1202).

IMO these words are ambiguous a little, some recent Python3 code still unnecessarily use `& 0xffffffff`:
https://github.com/search?l=Python&o=desc&q=python3+crc32+%26+0xffffffff&s=&type=Code

How about change like this, it's clearer.
```diff
    .. versionchanged:: 3.0
-      The result is always unsigned.
-      To generate the same numeric value across all Python versions and
-      platforms, use ``crc32(data) & 0xffffffff``.
+      Now the result is always unsigned.
+      In Python 2, to generate the same numeric value across all Python
+      versions and platforms, need to use ``crc32(data) & 0xffffffff``.
```

Reply:
i committed updates with more clear wording.

Reply:
This is what I wanted to fix, the previous `versionchange`s were somewhat misleading.